src/main/java/com/napier/sem/DAO.java

```diff
@@ -85,15 +129,22 @@ public DAO(Connection connection)
                 "        AND ci.id = co.capital\n" +
                 "    ) AS capital\n" +
                 "FROM country co\n" +
-                "WHERE co.continent = '"+continentName+"'\n" +
+                "WHERE co.continent = '" + continentName + "'\n" +
+                "AND co.population > 0\n " +
```

Comment:
nice one, consistent spacing is pleasing on the eye, and good catch with the populations